src/pages/Beers.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import Header from '../components/Header'
+import axios from 'axios'
+import {useState, useEffect} from 'react'
+import { Link } from 'react-router-dom'
+import Searchbar from '../components/Searchbar'
+
+const apiURL = "https://ih-beers-api2.herokuapp.com/beers"
+
+function Beers() {
+    const [beers, setBeers] = useState([])
+    const [searchBeers, setSearchBeers] = useState([])
+    const [showBeers, setShowBeers] = useState([])
+
+    const getBeers = async() =>{
+        try {
+            let response = await axios.get(apiURL)
+            setBeers(response.data)
+
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    useEffect(() => {
+        getBeers()
+    }, [])
+    
+    const searchBeersList = (query) => {
+        let searchedBeers = beers.filter((item) =>{
```

Comment:
Your `searchBeerList` function is not working because:
- You never set `searchBeers` with the api response (so searchBeers is an empty array);
- You're mapping through `beers` to display the list of beers so even if the function was working it would still not show the beer you searched for.

So to make it work:
- In the `getBeers` function you're setting `beers` but you need to set  `searchBeers` as well with `setSearchBeers(response.data)`
- In the `searchBeersList` function instead of `setShowBeers(searchedBeers)` do `setSearchBeers(searchedBeers)`
- In the return instead of looping through `beers` to display the list of beers loop through `searchBeers` ` {searchBeers.map((item) => {`